storybook/pages/CommunityTokenViewPage.qml

```diff
@@ -111,14 +111,14 @@ SplitView {
                             id: isAssetRadioButton
 
                             readonly property int type:
-                                TokenObject.Type.Asset
+                                Constants.TokenType.ERC20
```

Comment:
Pls add a comment to `Constants.TokenType` what is what :) I liked the old naming better, more descriptive

Reply:
In fact `ERC20` and `ERC721` are more technical, strict names. `Assets` and `Collectibles` are more our own "UI" names.